dev_guide/migrating_applications/web_framework_applications.adoc

```diff
@@ -271,33 +295,50 @@ $ oc new-app https://github.com/<github-id>/<repo-name>.git
 
 [[dev-guide-migrating-web-framework-applications-supported-Node.js-versions]]
 *Supported Node.js Versions*
-
+ifdef::openshift-enterprise,openshift-origin[]
 [cols="2,2",options="header"]
 |===
 |v2 |v3
 
-|Node.js 0.10  |Nodejs: 0.10, 4, and 6
+|Node.js 0.10  |link:https://www.openshift.com/features/containers.html[Supported Container Images]
 
 |    |Nodejs-mongodb-example. This quickstart template only supports Node.js version 6.
 
 |===
+endif::[]
 ifdef::openshift-online[]
+See link:https://www.openshift.com/features/containers.html#online3[Supported Container Images].
+
 [IMPORTANT]
 ====
 In {product-title} v3, version 0.10 is deprecated and no longer available to use.
 ====
 endif::openshift-online[]
-
+ifdef::openshift-dedicated[]
+See link:https://www.openshift.com/features/containers.html#dedicated[Supported Container Images]
+endif::[]
 [[dev-guide-migrating-web-framework-applications-wordpress]]
 == WordPress
 
+[IMPORTANT]
+====
+Currently, support for migrating WordPress applications is offered by the
+community only and not by Red Hat support.
+====
+
 For guidance on migrating WordPress applications to {product-title} v3, see the
 link:https://blog.openshift.com/migrating-wordpress-openshift-3/[OpenShift
 blog].
```

Comment:
So this was just posted (https://blog.openshift.com/running-wordpress-easy-way/)

@luciddreamz Thoughts on including the post here as a "for new wordpress applications see https://blog.openshift.com/running-wordpress-easy-way/"? Or leave no mention of it?

Reply:
Probably don't mention it. If Graham ever publishes his new [blog article](https://docs.google.com/document/d/1lXoQ2XjdSwCXff-0DhVpDHhFFwbtl_U-fd04JajC-nM/edit) and [Quickstart](https://github.com/openshift-evangelists/wordpress-quickstart), we should link to that, but probably not to this one.

Reply:
Should we even be claiming support for OpenShift Online? We in CEE don't directly support Wordpress (as we don't have a RH provided Wordpress image (or template), just community driven items.

Reply:
@sferich888 I think that's the intention of adding the note here, to call out that RH does *not* support Wordpress

Reply:
Correct. Those notes address https://bugzilla.redhat.com/show_bug.cgi?id=1476927